### DISCUSSION TITLE
Angular Reactive Form Knob

### DISCUSSION THREAD

**Discussion by Hugmungus** [storybookjs/storybook#11566] [Help]
Hi everyone, I'm really excited to get this setup in our project but I did have one question. I haven't seen a great example on how to setup a storybook knob that binds to a form control value. If anyone has any resources they could point me toward I would appreciate it. Thanks all!

**Comment by cuznerdexter** [comment]
Well it is 1year since the last post.
I am also looking for a way to use Reactive Form elements with Storybook.
It is a great tool, until you need to use a basic reactive form.  Does anyone have any way to use FormGroups / FormArrays / FormControls in Storybook ?
